cpp/tests/experimental/bfs_test.cpp

```diff
@@ -312,24 +265,45 @@ class Tests_BFS : public ::testing::TestWithParam<BFS_Usecase> {
   }
 };
 
+using cugraph::test::File_Usecase;
+using cugraph::test::Rmat_Usecase;
+
+using Tests_BFS_File = Tests_BFS<File_Usecase>;
+using Tests_BFS_Rmat = Tests_BFS<Rmat_Usecase>;
```

Comment:
Do we really need all these using statements?

Reply:
The first ones are for convenience, we could:
* Leave them out and always fully qualify references to the cugraph::test namespace
* Since these are all tests, we could just specify `using cugraph::test` and get anything in the test namespace

I'm fine with either.

The second pair is necessary for google test.  Google test uses macros to construct the test code below (`TEST_P`).  We can't specify a `Tests_BFS<File_Usecase>` as the first parameter to the macro.  I tried a number of variations to work around this, but it wants to construct a preprocessor token and can't include '_', '>' or ')' (not sure what else) in doing that.  The using statement redefines `Tests_BFS<File_Usecase>` or `Tests_BFS<cugraph::test::File_Usecase>` if we prefer into a token that google test can process through its macro expansion.